lib/util/theme_utils.dart

```diff
@@ -22,7 +22,7 @@ class ThemeUtils {
 
   /// this function is also called while building the button, so make sure we don't use any fallback
   /// to ensure the style reverts to the button theming
-  static ButtonStyle getButtonStyle({required bool isOutline, Color? backgroundColor, Color? color, RoundedRectangleBorder? border, EdgeInsets? padding, FontWeight? fontWeight, int? fontSize}) {
+  static ButtonStyle getButtonStyle({required bool isOutline, Color? backgroundColor, Color? color, RoundedRectangleBorder? border, EdgeInsets? padding, FontWeight? fontWeight, int? fontSize,double? buttonWidth,double? buttonHeight}) {
```

Comment:
use integer for width/height as that is what the user enters. You can convert to double when you need to `buttonWidth?.toDouble() ?? ...`